### PR TITLE
Change activity copy to use proper sentences

### PR DESCRIPTION
**Pull request by AndrewVos**
Also, add icons for each activity.

image

### CODE REVIEW

File: app/jobs/appointment_reminder_job.rb [hunk 1]
```diff
@@ -5,7 +5,7 @@ def perform(appointment)
     ReminderActivity.create!(
       appointment: appointment,
       owner: appointment.guider,
-      message: 'sent a reminder'
+      message: 'sent an appointment reminder email to the customer'
```
Comment (benlovell) [inline app/jobs/appointment_reminder_job.rb]: Maybe just worth keeping this as it was (maybe we should have just provided an empty string for `message` from the start) and putting this message directly in the template instead? My view on these is it's more important to use `message` when computing the message at runtime is potentially expensive - such as when we compute the audited fields.

Comment (AndrewVos) [inline app/jobs/appointment_reminder_job.rb]: Absolutely. It scares me that some of this is hardcoded and some stored in the DB.

Comment (AndrewVos) [inline app/jobs/appointment_reminder_job.rb]: I'm going to go through this code and remove the dependency on the `message` column, where appropriate.

File: app/jobs/appointment_reminder_job.rb [hunk 2]
```diff
@@ -4,8 +4,7 @@ class AppointmentReminderJob < ApplicationJob
   def perform(appointment)
     ReminderActivity.create!(
       appointment: appointment,
-      owner: appointment.guider,
-      message: 'sent a reminder'
```
Comment (benlovell) [inline app/jobs/appointment_reminder_job.rb]: For completion sake should we change the existing messages for this activity to `''`?

Comment (AndrewVos) [inline app/jobs/appointment_reminder_job.rb]: This is old code you're looking at surely

Comment (AndrewVos) [inline app/jobs/appointment_reminder_job.rb]: Oh disregard that I see what you mean. Message is defaulting to '' anyway so no reason for that.

Comment (AndrewVos) [inline app/jobs/appointment_reminder_job.rb]: You mean in the DB right?

Comment (benlovell) [inline app/jobs/appointment_reminder_job.rb]: Yep.

Comment (AndrewVos) [inline app/jobs/appointment_reminder_job.rb]: I would rather not, out of fear

**Pull request merged**